src/home_robot_hw/home_robot_hw/ros/grasp_helper.py

```diff
@@ -104,19 +104,7 @@ def msg_to_poses(self, msg, frame=None):
     def _score_cb(self, msg):
         with self.score_lock:
             self.score_id = msg.header.seq
-            self.scores = np.array([pt.x for x in msg.polygon.points])
-
-    def segmented_point_cloud_to_msg(self, xyz, labels):
```

Comment:
why is this deleted?

Reply:
Two `segmented_point_cloud_to_msg` were defined in the same class. Only one seems to be used (the one with both rgb and xyz as inputs), so I removed the other one.